Bangazon/Managers/PaymentTypeManager.cs

```diff
@@ -6,14 +6,18 @@ namespace BangazonCLI
     public class PaymentTypeManager
     {
         private dbUtilities _db;
+        private CustomerManager _cm;
         // Constructor method to establish a connection with the database, database conenction is passed in as an argument..
         public PaymentTypeManager(dbUtilities db)
         {
+            _cm = new CustomerManager(db);
             _db = db;
         }
         // Adds a new PaymentType--passed in as an argument--to the database
-        public int AddNewPaymentType(PaymentType newPaymentType){
-            return 4;
+        public int AddNewPaymentType(int newAccountNumber, string newAccountName){
```

Comment:
I think I'm with Ben on this - for each method's Add pass in the full object rather than individual properties. Whatever is going to call the AddNewPaymentType method will build up the entire object and pass in a PaymentType object as an argument.